scout/server/extensions/loqus_extension.py

```diff
@@ -99,7 +100,7 @@ def init_app(self, app):
 
     def version_check(self, loqusdb_settings):
         """Check if a compatible version is used otherwise raise an error"""
-        if not loqusdb_settings["version"] >= 2.5:
+        if not loqusdb_settings["version"] >= "2.5":
```

Comment:
The string comparison should work as expected, as long as loqus returns string-formatted numbers. If by chance it starts returning something like "v1.4" then we would be in trouble, because "v1.4" >= "2.5". But let's leave it like this for now!